File: 1. Two Sum/TwoSum.py
def twoSum(nums: list[int], target: int) -> list[int]:
    """
    :type nums: List[int]
    :type target: int
    :rtype: List[int]
    """

    for i in range(len(nums)):
        if target - nums[i] in nums and target - nums[i] != nums[i]:
            return [i, nums.index(target - nums[i])]

File: 1. Two Sum/TwoSum.js
function twoSum(nums, target) {
  for (let i = 0; i <= nums.length; i++) {
    if (target - nums[i] in nums && target - nums[i] != nums[i]) {
      return [nums.indexOf(target - nums[i]), i];
    }
  }
}
